<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>591</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="connectTab">
     <attribute name="title">
      <string>Connect to Server</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="Connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="checkinTab">
     <attribute name="title">
      <string>CheckIn</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="checkInLabel">
         <property name="text">
          <string>Client Storage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="CheckInlistWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="checkinButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>CheckIn File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkinRefresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="ckeckoutTab">
     <attribute name="title">
      <string>CheckOut</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="checkoutLabel">
         <property name="text">
          <string>Server Storage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="checkoutlistWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkoutpushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>CheckOut File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkoutRefresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="ViewFile">
     <attribute name="title">
      <string>View File</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="viewServerStorage">
         <property name="text">
          <string>Server Storage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="viewfilelistWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="viewFileButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>View File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viefilerefresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="repositorycontent">
     <attribute name="title">
      <string>Repository Contents</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>258</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="repoContentServerStorage">
         <property name="text">
          <string>Server Storage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="repoContent">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>258</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="repoContentClientStorage">
         <property name="text">
          <string>Client Storage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="clientStorageRepoContent"/>
       </item>
       <item>
        <widget class="QPushButton" name="repoStorageClientStorage">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="deleteTab">
     <attribute name="title">
      <string>Delete</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="deleteServerStorage">
         <property name="text">
          <string>Server Storage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="deleteListWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteRefresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
